src/borg/archive.py

```diff
@@ -1289,6 +1292,24 @@ def verify_file_chunks(item):
                 logger.info('{}: Completely healed previously damaged file!'.format(item.path))
                 del item.chunks_healthy
             item.chunks = chunk_list
+            if 'size' in item:
+                item_size = item.size
+                item_chunks_size = item.get_size(compressed=False, from_chunks=True, from_chunks_healthy=False)
+                if __name__ == '__main__':
+                    if item_size != item_chunks_size:
+                        if has_chunks_healthy:
+                            item_chunks_healthy_size = item.get_size(compressed=False,
+                                                                     from_chunks=False, from_chunks_healthy=True)
```

Comment:
hmm, we can't get anything useful from chunks_healthy list. if a chunkid is in there, but not in chunks list, we have lost that chunk, so we can't determine its size. so, this is not implementable.